app/Models/projects.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class projects extends Model
```

Comment:
File and Class name should start in upper case, MUST start with a noun (in singular form) `Project`